lib/entities.js

```diff
@@ -244,4 +241,4 @@ exports.entityMap = {
        hearts: "♥",
        diams: "♦"
 };
-//for(var  n in exports.entityMap){console.log(exports.entityMap[n].charCodeAt())}
+//for(var  n in exports.entityMap){console.log(exports.entityMap[n].charCodeAt())}
```

Comment:
This change which reverts some formatting cleanup is something I don't really like about reverts in general.

This line looks pretty useless. I just raised PR #85 to completely remove it.

Reply:
Looks good, how do you want me to handle this PR?

Reply:
Easy way to resolve the conflict will be for me to drop this hunk, which I just did.